packages/react-composites/src/composites/common/ParticipantContainer.tsx

```diff
@@ -70,6 +72,7 @@ export const ParticipantListWithHeading = (props: {
             />
           )}
           onFetchParticipantMenuItems={onFetchParticipantMenuItems}
+          onParticipantClick={props.onParticipantClick}
```

Comment:
Can remove this line as onParticipantList should be part of `{...participantListProps}` (when [other comment](https://github.com/Azure/communication-ui-library/pull/1515/files#r812139544) is resolved)